docs/pages/en-us/Architecture.md

```diff
@@ -13,26 +13,48 @@ banner:
 
 # Architecture
 
-The Audit library logs audit event metadata, belonging to tenant applications, into Elasticsearch.
+The Audit library logs audit event metadata, belonging to tenant applications, into Elasticsearch directly or via the Audit Web Service HTTP REST API.
```

Comment:
Replace all instances of 'Audit Web Service HTTP REST API' with 'Audit Web Service REST API' as HTTP is implied with REST.